code/modules/power/supermatter/supermatter.dm

```diff
@@ -1025,6 +1035,27 @@ GLOBAL_DATUM(main_supermatter_engine, /obj/machinery/power/supermatter_crystal)
 	COOLDOWN_START(src, common_radio_cooldown, SUPERMATTER_COMMON_RADIO_DELAY)
 	return TRUE
 
+/obj/machinery/power/supermatter_crystal/proc/holiday_lights()
+	desc += "\nRadiating both festive cheer and actual radiation, it has a dazzling spectacle lights wrapped lovingly around the base transforming it from a potential doomsday device into a cosmic yuletide centerpiece."
```

Comment:
Same advice here--I think this should be checked in `examine` rather than added to desc.